package.json

```diff
@@ -36,7 +38,7 @@
     ]
```

Comment:
### Your Readme :memo:
![image](https://user-images.githubusercontent.com/93039359/201008260-26311090-40af-4dca-ab05-8d1ce7ea9473.png)

- [ ] Your readme file is not descriptive means Live Demo Link, Authors section should have clickable links to your social media accounts, Getting Started Section, License etc have not been properly showed. It would be nice if you would make your readme look proper and descriptive too :blush:

### Hint :green_circle:
You can see this [readme](https://github.com/HAMMAS-SALEEM/Math-Magician#math-magician) file to get an understanding of what i am talking about :blush: